src/utils/dateUtils/dateCalculations.js

```diff
@@ -1,4 +1,12 @@
 export const calculateDateDaysInPast = (date, days) => {
```

Comment:
To reduce a teensy bit of duplication and (maybe) improve readability, what do you think of creating a helper function:
```javascript
/** Converts an input to a number if possible; if fails, returns null */
function parseIntExact(val) {
  if (typeof val === 'number') return val;

  // need to check isNaN(num) because parseInt turns '12hello' into a number
  if (isNaN(val)) return null;

  return parseInt(val);
}
```
I don't think `isNaN(parsedVal)` is necessary because you checked isNaN already.  You can add it back if I'm wrong.

Also, calculateDateDaysInPast, calculateDateMonthsInPast, & calculateDateDaysInPastFromToday should be rewritten with `function` syntax :)